### ISSUE TITLE
R data download doesn’t work

### ISSUE THREAD

**Issue by CB-HAL** [IQSS/dataverse#9490]
Tabular data as R data download does not function. Data files are correct ingested, all kind of original formats are affected (Stata, SPSS, csv):
status: "ERROR”
code: 404
message: "datafile access error: requested optional service (image scaling, format conversion, etc.) could not be performed on this datafile."

server.log:
[2023-03-31T14:35:01.873+0200] [Payara 5.2022.5] [SEVERE] [] [edu.harvard.iq.dataverse.rserve] [tid: _ThreadID=94 _ThreadName=http-thread-pool::jk-connector(1)] [timeMillis: 168026>
  voidEval failed, request status: error code: 127]]

e.g. 
https://data.aussda.at/api/access/datafile/5991?format=RData (original Stata)
https://data.aussda.at/api/access/datafile/4433?format=RData (original csv)
Not all files are affected, a few downloads are ok, e.g.:
https://data.aussda.at/api/access/datafile/4436?format=RData (Stata)
There is no identifiable pattern as to which ones work and which ones don't.
All settings seem to be ok https://guides.dataverse.org/en/5.13/installation/prerequisites.html#r 
Dataverse 5.12.1

Other instances have the same error:
https://dataverse.harvard.edu/api/access/datafile/7029119?format=RData 
https://dataverse.harvard.edu/api/access/datafile/6693766?format=RData
https://www.sodha.be/api/access/datafile/2487?format=RData&gbrecs=true

**Comment by landreev** [comment]
Unfortunately, this download-as-RData support, that uses a remote R instance via Rserve, is just flaky and unreliable. The whole subsystem is rather obsolete by now, and we are seriously considering retiring it. 
There's some lively debate (including in the Dataverse users group right now) about whether this "download-as-RData" functionality is actually providing any useful value. (If the ingested original was a Stata file, any R user can easily download the .dta file and import it into R - which has excellent support for Stata via the package "foreign"; if the original was RData... the whole point is mute; etc. etc.)

**Comment by CB-HAL** [comment]
I also think that R users can handle other data formats. How can I deactivate the R download, only uninstall R and remove the jvm-options?

**Comment by landreev** [comment]
> I also think that R users can handle other data formats. How can I deactivate the R download, only uninstall R and remove the jvm-options?

Unfortunately, there's no way to deactivate it currently. (i.e., if you uninstall the components, the download link will still be there, but it will be producing the "access error" for all tab. files. 

It would of course be **very** easy to make it possible to disable it; I'll try to prioritize it.

**Comment by CB-HAL** [comment]
fine, thank you

**Comment by landreev** [comment]
Yes, I'll push for prioritizing #8711 - should be very straightforward.

**Comment by CB-HAL** [comment]
@landreev this problem still exists and we have sometimes users reporting these problem to us. maybe it will be possible to put this on the agenda for one of the next releases.
